mbed_lib.json

```diff
@@ -14,7 +14,12 @@
         "fea-ipv6": {
             "help": "Used to globally disable ipv6 tracing features.",
             "value": null
+        },
+        "color-theme": {
+            "help": "Set color theme. 0 for readable, 1 for unobtrusive.",
+            "options": [0, 1],
+            "macro_name": "MBED_TRACE_COLOR_THEME",
```

Comment:
I think the default should be the one we had previously

Reply:
That would make it different from the default in Mbed OS. 

The main reason for this change, and to have the default be more visible, is to help our FAEs (and me) when our customers send us screenshots of their logs or do screen sharing during remote debug sessions, so we can actually read the debug output and help our customers.